packages/svelte/tests/css/samples/global-with-nesting/input.svelte

```diff
@@ -0,0 +1,10 @@
+<style>
+	div {
+		&:global(.class){
+			color: red;
+		}
+	}
+</style>
+
+<div>
+</div>
```

Comment:
Can we have another test for `<div><p class="class">foo</p></div>` to check that `class` isn't scoped in the HTML?

Reply:
Sure...but actually it needs to be a bit different i think because this cover the case for
```css
div{
    &.class{
    }
}
```
which is where the class is attached to the div.

Reply:
Added now!

Reply:
You're right, doh - thanks!